_common.sh

```diff
@@ -266,8 +266,13 @@ function start_tomcat {
 
 	"./${TEMP_DIR}/liferay/tomcat/bin/catalina.sh" start
 
-	until curl --fail --output /dev/null --silent http://localhost:8080
+	for count in {0..30}
 	do
+		if (curl --fail --max-time 3 --output /dev/null --silent http://localhost:8080)
+		then
+			break
+		fi
+
```

Comment:
@kiwm, does it make sense we try to stop a tomcat which wasn't able to be started? Can we improve it?